src/trainer_see.c

```diff
@@ -64,6 +64,8 @@ static const u8 sEmotion_QuestionMarkGfx[] = INCBIN_U8("graphics/field_effects/p
 static const u8 sEmotion_HeartGfx[] = INCBIN_U8("graphics/field_effects/pics/emotion_heart.4bpp");
 static const u8 sEmotion_DoubleExclamationMarkGfx[] = INCBIN_U8("graphics/field_effects/pics/emotion_double_exclamation.4bpp");
 static const u8 sEmotion_XGfx[] = INCBIN_U8("graphics/field_effects/pics/emote_x.4bpp");
+// TODO: Credit https://www.spriters-resource.com/ds_dsi/pokemonheartgoldsoulsilver/sheet/30497/
```

Comment:
What exactly does this todo entail?

Reply:
This is the initial spritesheet from which I ripped the Gen 1-3 follower sprites, from HGSS, so it would be nice to credit the person who made those somewhere,  I think.

Normally I leave `TODO: Credit` comments in my code so I can grep them for the eventual release of my hack

Reply:
Would it be ok for the expansion to just credit them here?